cmd/triggerRun/cmd/root.go

```diff
@@ -157,31 +179,60 @@ func processTriggerSpec(client triggersclientset.Interface, tri *triggersv1.Trig
 		return nil, errors.New("Trigger is not defined")
 	}
 
+	//convert trigger to eventListener
 	el, err := triggersv1.ToEventListenerTrigger(tri.Spec)
 	if err != nil {
 		return nil, fmt.Errorf("fail to convert Trigger to EvenetListener: %w", err)
 	}
 
 	log := eventLog.With(zap.String(triggersv1.TriggerLabelKey, el.Name))
 
+	config, err := clientcmd.BuildConfigFromFlags("", kubeconfig)
+	if err != nil {
+		log.Fatalf("Failed to get in cluster config: %v", err)
+	}
+
+	kubeClient, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		log.Fatalf("Failed to get the Kubernetes client set: %v", err)
+	}
+
+	r := sink.Sink{
+		KubeClientSet: kubeClient,
+		HTTPClient:    http.DefaultClient,
+		Auth:          sink.DefaultAuthOverride{},
+	}
```

Comment:
Some of this logic is shared with the triggersclientset creation (e.g. reading the kubeconfig). Can we consolidate these together?